grpc-proto/proto/query.proto

```diff
@@ -57,9 +57,11 @@ message UdtValue {
 
 // A UUID value (for CQL types uuid and timeuuid).
 message Uuid {
-  // TODO: Is this going to have endian problems representing it this way?
-  fixed64 msb = 1;
-  fixed64 lsb = 2;
+  bytes value = 1;
+}
+
+message Inet {
```

Comment:
Would it make sense to add a comment that it's 4 or 16 bytes (ipv4 vs 6)?

Reply:
Good idea. thanks